app/models/member.rb

```diff
@@ -51,6 +53,18 @@ def first_name
   def self.active
     Member.where(skip_meetings: false)
   end
+
+  def activate_account
+    self.email_confirmed = true
+    self.email_confirmation_token = nil
+    save!
+  end
+
+  private
+
+  def assign_email_confirmation_token
```

Comment:
This is a misleading name, you only assign a confirmation token when there isn't one, I would rename to `ensure_email_confirmation_token`.

A good practice here is to `return` instead of doing a right-hand-side `if`.

```ruby
def assign_email_confirmation_token
   return unless email_confirmation_token.blank?
   self.email_...
end
```